src/apps/api/urls.py

```diff
@@ -19,7 +19,8 @@
 router.register('leaderboards', leaderboards.LeaderboardViewSet)
 router.register('submission_scores', leaderboards.SubmissionScoreViewSet, 'submission_scores')
 router.register('tasks', tasks.TaskViewSet)
-router.register('tasksearch', tasks.TaskViewSetSimple)
+# TODO: We don't need a whole separate viewset to filter tasks, also messes up reverse searching 'task-list'
+# router.register('tasksearch', tasks.TaskViewSetSimple)
```

Comment:
This is used for searching tasks on comp editor phase modal. Removing this viewset will break that, we could maybe implement this behavior on an action? but the easier fix (for now) is to add a basename to the router.register:
```python
router.register('tasksearch', tasks.TaskViewSetSimple, base_name='tasksearch')
```
and then, at the very least, you won't get `reverse` issues, and task searching will still work.

Reply:
I feel like a normal `GET` to `/tasks` should be the "search?" not necessary to have an action or something?

Reply:
they require different serializers, how are we going to pass a different serializer for the different get requests?

Reply:
I believe we could make a "simple list serializer" that works for both a simple "get" and a "get with search params?" It may return a bit more data than you need in some cases, but that should be fine?

Reply:
I guess, though we are changing the names of one of the fields in the simple serializer `key -> value` so that it will work w/ the multiselect. I guess we could just return the key twice? once as key, once as value?